src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -692,13 +695,45 @@ export const duplicateForm = (
     overrideProps.endPage = omit(originalForm.endPage, 'buttonLink')
   }
 
+  // If the form is an MRF, copy the workflow as well
```

Comment:
suggestion: wonder if we could include some unit tests for the service to cover this new duplication behavior as well? 

in particular, these 3 tcs: 
- for static workflow step, replaces with overrideEmails if exist and use old if not exist 
- for conditional, dynamic: uses old regardless if the overrideEmails exist 
- for dropdown field, if overrideEmails exist, will replace. otherwise, use existing. 

this would give us confidence that mrf workflow duplication will not regress when making future changes